src/misc/cache/lv_image_cache.c

```diff
@@ -37,6 +49,38 @@
 /**********************
  *   GLOBAL FUNCTIONS
  **********************/
+
+bool lv_image_cache_init(void)
+{
+#if LV_CACHE_DEF_SIZE > 0
+    img_cache_p = lv_cache_create(&lv_cache_class_lru_rb_size,
+    sizeof(lv_image_cache_data_t), LV_CACHE_DEF_SIZE, (lv_cache_ops_t) {
+        .compare_cb = (lv_cache_compare_cb_t) image_cache_compare_cb,
+        .create_cb = NULL,
+        .free_cb = (lv_cache_free_cb_t) image_cache_free_cb,
+    });
+    return img_cache_p != NULL;
+#else
+    return false;
+#endif
+}
+
+bool lv_image_header_cache_init(void)
+{
+#if LV_IMAGE_HEADER_CACHE_DEF_CNT > 0
+    img_header_cache_p = lv_cache_create(&lv_cache_class_lru_rb_count,
+    sizeof(lv_image_header_cache_data_t), LV_IMAGE_HEADER_CACHE_DEF_CNT, (lv_cache_ops_t) {
+        .compare_cb = (lv_cache_compare_cb_t) image_header_cache_compare_cb,
+        .create_cb = NULL,
+        .free_cb = (lv_cache_free_cb_t) image_header_cache_free_cb
+    });
+
+    return img_header_cache_p != NULL;
+#else
+    return false;
+#endif
+}
```

Comment:
Would it make sense to add `if(img_cache_p != NULL) return true;` and `if(img_header_cache_p != NULL) return true;` since these are functions are part of a public API?

Reply:
In fact, it should not be called by other functions, but your suggestion is very good. I'll add it!

Reply:
Done